### ISSUE TITLE
Have "quick stats" shown near the graphs for year-round info

### ISSUE THREAD

**Issue by Louis-Backstrom** [bird-team/brisbane-bird-atlas#116]
I think it'd be useful to have some year-round stats associated with the graphs for people to see. The overall reporting rate is present up the top section, but things like average elevation and average count across the entire period aren't actually clearly shown - you can get a rough guess by averaging the mean lines of each month but that's not particularly useful. I'm guessing it should be a relatively easy thing to add - average count especially is just a quick pull from the data excel (I'm not actually really sure how we get elevational data, but I assume if you can do it for each month you could do it across the entire period)

@jeffreyhanson does something like this sound doable?

**Comment by jeffreyhanson** [comment]
Yeah, the code for calculating the average is fairly simple, and you're right that we already have the data to it do so since it's used in the graphs. The main issue is making it scalable so it doesn't increase build time beyond TravisCI's and CircleCI's limits - which generally means that a lot of extra code is needed (e.g. to export the info as an asset, like the figures and tables). Where would be the best place to display the information - in the existing table with the species summary? Or in a new table? Or as text in the species accounts?

**Comment by Louis-Backstrom** [comment]
I think below the graphs makes the most sense - that's where the monthly info is so it makes sense to me to have the overall info for the same there too. Given that then, is it possible to add this in as part of the graph assets or would they have to be separate (and just displayed one after the other)?

I was thinking underneath - like this (excusing the crude drawing skills)

![image](https://user-images.githubusercontent.com/44354406/52519843-68c1b380-2cad-11e9-88d9-67092a66000f.png)

**Comment by jeffreyhanson** [comment]
Sorry for the slow response. It's possible to include that in the graph - but tricky. I think it might be simpler to add this as a new table? But that would require a fair amount of work.

**Comment by dbl3raf** [comment]
Thanks Jeff for your assessment - while it is a good idea, I think it is too much of a burden on your time at the moment and perhaps we could hibernate this issue for a bit

**Comment by Louis-Backstrom** [comment]
Yep, more than happy to hibernate this issue too - I'm just adding suggestions here as I think of them so if we have time later on we can go through and add them into the atlas.

**Comment by jeffreyhanson** [comment]
Ok great. Yeah, I think it's good idea to write up and discuss as many ideas as possible so they don't get lost or forgotten. Also, this way we'll have a list of potential features to add to atlas for when I find more time to work on the atlas.
